app/helpers/dor_object_helper.rb

```diff
@@ -96,16 +96,6 @@ def last_accessioned_version(object)
     Sdr::Client.current_version(object.pid)
   end
 
-  ##
-  # @deprecated Please use non-blocking requests rather than blocking helpers.
```

Comment:
:+1: It's definitely less confusing for a blocking request to happen in a model.